R/publications.r

```diff
@@ -35,6 +35,9 @@ utils::globalVariables(c("."))
 ##' @export
 get_publications <- function(id, cstart = 0, pagesize=100, flush=FALSE) {
 
+    ## Make sure pagesize is not greater than max allowed by Google Scholar
+    if (pagesize > 100) pagesize <- 100
+
```

Comment:
Could I suggest that you add a warning when changing the `pagesize` arguments? e.g.
```r
if (pagesize > 100) {
  warning("pagesize: ", pagesize, " exceeds Google Scholar maximum. Setting to 100.")
  pagesize <- 100
}

Reply:
Yes, it's a great suggestion! Will make the recommended change